scipy/signal/_filter_design.py

```diff
@@ -5310,7 +5327,7 @@ def besselap(N, norm='phase', *, xp=None, device=None):
             float(k))
 
 
-def iirnotch(w0, Q, fs=2.0):
+def iirnotch(w0, Q, fs=2.0, *, xp=None, device=None):
```

Comment:
Can we add docs for `xp` and `device` like in the other prs?

Reply:
Thanks, done, here and for other functions touched here.